tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -226,46 +230,46 @@ private void registerExecutionResult(int exitCode, String jobName, String jobInd
     }
   }
 
-  private class Heartbeater implements Runnable {
-    int hbMissCounter = 0;
-    int numHbToMiss;
+private class Heartbeater implements Runnable {
```

Comment:
I preferred the previous indentation. I think nested classes should be indented to emphasize they are contained inside another class.

Reply:
hmm, I did a copy all & paste to use the format suggested by Intellij. Let me double check the indentation for this blob.